tests/test_exporters.py

```diff
@@ -299,11 +299,11 @@ def test_stl_ascii(tmpdir, box123, id, opt, matchvals):
 @pytest.mark.parametrize(
     "id, opt, matchval",
     [
-        (0, None, b"STL Exported by OpenCASCADE"),
-        (1, {"ascii": False}, b"STL Exported by OpenCASCADE"),
-        (2, {"ASCII": False}, b"STL Exported by OpenCASCADE"),
-        (3, {"unknown_opt": 1}, b"STL Exported by OpenCASCADE"),
-        (4, {"unknown_opt": 1, "ascii": False}, b"STL Exported by OpenCASCADE"),
+        (0, None, b"STL Exported by Open CASCADE"),
```

Comment:
I would have thought it was correct without the space. It's a minor difference though.

Reply:
Looks like something changed on the OCCT side.